data/data/aws/bootstrap/variables.tf

```diff
@@ -3,9 +3,9 @@ variable "ami" {
   description = "The AMI ID for the bootstrap node."
 }
 
-variable "cluster_name" {
+variable "cluster_id" {
   type        = "string"
-  description = "The name of the cluster."
+  description = "The identifier for the cluster."
```

Comment:
Can we add human-readable to the description to increase the differentiation between the UUID?

Reply:
Why should we differentiate between UUID and HumanID in terraform?

Reply:
Within terraform we do not need to differentiate. However, this is at the boundary between terraform and go. The go code is ambiguous with the term cluster ID. As a reader of the code, it is not immediately clear to me here whether the cluster ID that terraform is expecting is the UUID or the human ID. We could either (1) expect the reader of the code to dig further into the code to figure out which one is expected here or (2) add more information to the description of the variable to make it clear.

Reply:
the boundary says give me an identifier for the cluster, it doesn't need to care if it was given an UUID or a ID that is human readable... ?

Let me update it to include constraints like length, characters,  because that's what it cares any nothing more.

Reply:
https://github.com/openshift/installer/pull/1280/commits/e0614f224730024ea22c3e4edee053f617970739